metaflow/task.py

```diff
@@ -280,25 +283,42 @@ def clone_only(
                 "task.clone_only needs a valid clone_origin_task value."
             )
         origin_run_id, _, origin_task_id = clone_origin_task.split("/")
+        # Update system logger context
+        # We don't update the context for the system monitor as the cardinality of this context is high
+        _system_logger.update_context(
+            {
+                "run_id": run_id,
+                "step_name": step_name,
+                "task_id": task_id,
+                "retry_count": retry_count,
+                "project_name": current.get("project_name"),
+                "branch_name": current.get("branch_name"),
+                "is_user_branch": current.get("is_user_branch"),
+                "is_production": current.get("is_production"),
+                "project_flow_name": current.get("project_flow_name"),
+                "origin_run_id": origin_run_id,
+                "origin_task_id": origin_task_id,
+                "trace_id": trace_id or None,
+            }
+        )
 
-        msg = {
-            "task_id": task_id,
-            "msg": "Cloning task from {}/{}/{}/{} to {}/{}/{}/{}".format(
-                self.flow.name,
-                origin_run_id,
-                step_name,
-                origin_task_id,
-                self.flow.name,
-                run_id,
-                step_name,
-                task_id,
-            ),
-            "step_name": step_name,
-            "run_id": run_id,
-            "flow_name": self.flow.name,
-            "ts": round(time.time()),
-        }
-        self.event_logger.log(msg)
```

Comment:
curious, what is the impact of still including `self.event_logger.log` if we chose to? would it break anything on your end downstream?